### PR TITLE
Update Doc for Kinesis log sourcece

### PR DESCRIPTION
**Pull request by Gourav2906**
solves https://github.com/SumoLogic/terraform-provider-sumologic/issues/192

### CODE REVIEW

**Review by vsinghal13 — APPROVED** [review]

File: website/docs/r/http_source.html.markdown [hunk 1]
```diff
@@ -32,6 +32,14 @@ resource "sumologic_http_source" "http_traces_source" {
   content_type        = "Zipkin"
 }
 
+resource "sumologic_http_source" "kinesisLog" {
+  name = "demo-name"
+  description = demo-desc"
```
Comment (maimaisie) [inline website/docs/r/http_source.html.markdown:37]: missing opening quotes

File: website/docs/r/http_source.html.markdown [hunk 2]
```diff
@@ -32,6 +32,14 @@ resource "sumologic_http_source" "http_traces_source" {
...
+  category = "demo-desc"
```
Comment (maimaisie) [inline website/docs/r/http_source.html.markdown:38]: nit: `"demo-category"`

File: website/docs/r/http_source.html.markdown [hunk 3]
```diff
@@ -43,7 +51,7 @@ resource "sumologic_collector" "collector" {
 In addition to the common properties, the following arguments are supported:
 
 - `message_per_request` - (Optional) When set to `true`, will create one log message per HTTP request.
-- `content_type`        - (Optional) When configuring a HTTP Traces Source, set this property to `Zipkin`. This should only be used when creating a Traces source.
+- `content_type`        - (Optional) When configuring a HTTP Traces Source, set this property to `Zipkin` and `KinesisLog` for Kinesis Log source. This should only be used when creating a Traces or Kinesis Log source.
```
Comment (maimaisie) [inline website/docs/r/http_source.html.markdown:54]: The phrasing is a bit confusing. Can we change to 
> When configuring an HTTP Traces Source, set this property to `Zipkin`. When configuring a Kinesis Log source, set this property to  `KinesisLog`.

Comment (Gourav2906) [reply website/docs/r/http_source.html.markdown]: Done

File: website/docs/r/http_source.html.markdown (hunk 2 above)
Comment (Gourav2906) [reply website/docs/r/http_source.html.markdown]: Done

File: website/docs/r/http_source.html.markdown (hunk 1 above)
Comment (Gourav2906) [reply website/docs/r/http_source.html.markdown]: Done

**Review by maimaisie — APPROVED** [review]

**Pull request merged**